packages/json-schema-worker/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "@cloudflare/json-schema-worker",
+	"version": "0.0.0",
+	"private": true,
+	"scripts": {
+		"deploy": "wrangler publish",
```

Comment:
You `npm run publish` from the Action workflow, but it's called `deploy` here.

Reply:
🤦